src/api/profile.ts

```diff
@@ -48,3 +48,40 @@ export const makeUpdateUserGridLocation = ({
       firestore.doc(doc).set(newData);
     });
 };
+
+export const makeUpdateUserOnlineStatus = ({
+  venueId,
+  userUid,
+}: MakeUpdateUserGridLocationProps) => (status: string) => {
+  const doc = `users/${userUid}`;
```

Comment:
Instead of using this 'path' format, we should use `firestore.collection('users').doc(userUid)` below.